Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM node:8.16.1-alpine
```

Comment:
Given this is a new repo we should probably be using Node 12 as the current version. We should really be looking to move all other services to this too and will need to do so before Jan 2020 when Node 8 maintenance finishes - https://nodejs.org/en/about/releases/

Reply:
Have updated that